Makefile

```diff
@@ -1,24 +1,35 @@
+include ./.bingo/Variables.mk
+
+# ADO does not expose an unauthenticated API for fetching one file, and the git server
+# does not have support for `git archive`, so we need to do some silly things to grab
+# just the OpenAPIv2 specification we need instead of the 10GiB in the rest of the repo...
+pkg/dataplane/internal/msi-credentials-data-plane.openapi.v2.json:
```

Comment:
do you want `.PHONY` targets for these, in case we change the path to be the new API version later down the road, or do you expect someone remove the file, update the makefile target with the new API version, and then make it?

Reply:
I specifically did not want `.PHONY` since that gets in the way of the "update the spec, regenerate the Go" flow I was using. I think yes they would update what version we want and `make --always-make` their way into the update.